recipe-server/normandy/selfrepair/views.py

```diff
@@ -4,11 +4,14 @@
 from django.shortcuts import render
 from django.views.decorators.cache import cache_control
 
+from csp.decorators import csp_replace
 from rest_framework.reverse import reverse
 
 from normandy.base.decorators import short_circuit_middlewares
 
 
+# This page is primarily used by a hidden iframe in Firefox, so allow framing.
+@csp_replace(FRAME_SRC='*', FRAME_ANCESTORS='*')
```

Comment:
I'm not sure what the origin should be. We aren't being iframed in the traditional sense, but we are being [loaded in a "hidden browser"](https://dxr.mozilla.org/mozilla-central/source/browser/modules/SelfSupportBackend.jsm?q=path%3Aself+ext%3Ajsm&redirect_type=single#255-269). I'd be happy to restrict the origin to only that, if we can figure out what it is.